packages/components/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,263 @@
+import { Pressable, StyleProp, Text, View, ViewStyle } from 'react-native'
+import { spacing } from '@department-of-veterans-affairs/mobile-tokens'
+import { useTranslation } from 'react-i18next'
+import React, { FC } from 'react'
+
+import { ComponentWrapper } from '../../wrapper'
+import { Icon } from '../Icon/Icon'
+import { Spacer } from '../Spacer/Spacer'
+import { useTheme } from '../../utils'
+
+export type CheckboxProps = {
+  /** Primary text for checkbox */
+  label: string
+  /** OnPress function. Pass a function that alters `checked` state */
```

Comment:
Reads as maybe too prescriptive documentation? Could also alter indeterminate and other things on the app level.

```suggestion
  /** OnPress logic to alter `checked` state or other behavior associated with the checkbox */
```
?